ansible/devutils

```diff
@@ -82,6 +129,87 @@ def action_console(parameters):
                                     console_password=vars['creds']['console_password'][vars['console_type']])
         console_host.posix_shell()
 
+
+def pdu_action_on_dut(host, attrs, action):
+    ret = { 'DUT' : host, 'PDU status' : [], 'Summary' : [], 'Action' : action }
+    pdu_name = attrs['pdu_host'] if 'pdu_host' in attrs else None
```

Comment:
If we need to support the case that a DUT is connected to two PDUs as we have talked offline, this logic has to be updated.

Reply:
Already supported. Please ignore.